integration-cli/docker_cli_inspect_test.go

```diff
@@ -69,7 +72,19 @@ func (s *DockerSuite) TestInspectImageFilterInt(c *check.C) {
 }
 
 func (s *DockerSuite) TestInspectContainerFilterInt(c *check.C) {
-	runCmd := exec.Command("bash", "-c", `echo "blahblah" | docker run -i -a stdin busybox cat`)
+	tmpDir, err := ioutil.TempDir("", "insp")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(tmpDir)
+
+	fn := filepath.Join(tmpDir, "inspTest")
+	err = ioutil.WriteFile(fn, []byte("blahblah"), 0777)
+	c.Assert(err, check.IsNil)
+
+	r, err := os.Open(fn)
+	c.Assert(err, check.IsNil)
+
+	runCmd := exec.Command(dockerBinary, "run", "-i", "-a", "stdin", "busybox", "cat")
+	runCmd.Stdin = r
```

Comment:
This test could be much simpler if you just use a string buffer and use that as stdin correct?